example/Makefile

```diff
@@ -0,0 +1,21 @@
+CC?=gcc
```

Comment:
Do you want to discuss the hierarchy of the project ? I find it a little bit confusing.
First, maybe we should prepare the possibility of multiple examples ? (like the future K one). We would have an `examples` folder instead of `example` and a few subfolders for each example. We can have a makefile compiling them to.
Also, the `main.c` in the `src` folder is kinda an example, should it be in this `examples` directory ? As the guests are related to it, maybe we should move them too ?

Reply:
Done 345de25